src/components/VoiceSearch/__tests__/__snapshots__/VoiceSearch-test.tsx.snap

```diff
@@ -2,38 +2,51 @@
 
 exports[`VoiceSearch Rendering with custom template for status 1`] = `
 <div
```

Comment:
how come this isn't wrapped in an array?

Reply:
https://github.com/algolia/instantsearch.js/pull/4156#discussion_r331562006